pymarlin/core/trainer.py

```diff
@@ -140,11 +143,39 @@ def __init__(
         self.trainer_backend.init(self._get_trainer_backend_args())
         self.trainer_backend.update_state(self.checkpointed_states.trainer_backend_state)
 
+        if self.args.ort: 
```

Comment:
@krishansubudhi while this is a small check and ORTModule integration is somewhat simple - do you think this extension should prompt a child class (ORTTrainer) instead of adding to Trainer?

Reply:
I think Trainer should be left unchanged. Trainer does rank  management, checkpointing and orchestrating. Don't think ORTModule changes any of that.

Reply:
Sure I can advise Felix on creating a ORTTrainerBackend with this logic there.